utils/helpers.ts

```diff
@@ -24,4 +27,216 @@ function formatTime(duration: number): string {
   return `About ${duration} Hours`;
 }
 
```

Comment:
You could perhaps use comments to specify that it's for a certain game

Reply:
Done.